### DISCUSSION TITLE
Forums

### DISCUSSION THREAD

**Discussion by putt1ck** [winder/Universal-G-Code-Sender#2628] [General]
Are the forums (Google Groups) abandoned or just temporarily disabled by Google? If they are no longer in use is there somewhere else to ask questions that the community (rather than devs) might be able to answer?

**Comment by breiler** [comment]
The Google Groups were infested with spammers and it was shut down (by Google) and we were never able to bring it up again.

There is this forum for discussions or this [Discord server](https://discord.gg/4DYywtyGYK) which we monitor.

You are also able to get help on Reddit (https://www.reddit.com/r/hobbycnc/) or on Facebook https://www.facebook.com/groups/713561279087507
